grafana/Dockerfile

```diff
@@ -1,13 +1,64 @@
+ARG GRAFANA_VERSION=7.5.16
```

Comment:
Do not specify a value here, it will be taken from the Makefile. If we have a default here as well, it will confuse when updating to a newer version